src/main/java/com/linkedin/kmf/apps/SingleClusterMonitor.java

```diff
@@ -260,8 +260,8 @@ public static void main(String[] args) throws Exception {
     if (res.getString("latencyPercentileGranularityMs") != null)
       props.put(ConsumeServiceConfig.LATENCY_PERCENTILE_GRANULARITY_MS_CONFIG, res.getString("latencyPercentileGranularityMs"));
 
-    SingleClusterMonitor test = new SingleClusterMonitor(props, "end-to-end");
-    test.start();
+    SingleClusterMonitor app = new SingleClusterMonitor(props, "end-to-end");
```

Comment:
@lindong28 Although kind of unrelated to this commit. I feel this name is better given the new hierarchy